second-edition/src/ch06-01-defining-an-enum.md

```diff
@@ -97,23 +102,31 @@ let loopback = IpAddr {
 };
 ```
 
-<span class="caption">Listing 6-1: Storing the data and `IpAddrKind` variant of
-an IP address using a `struct`</span>
-
-Here, we’ve defined a struct `IpAddr` that has two fields: a `kind` field that
-is of type `IpAddrKind` (the enum we defined previously) and an `address` field
-of type `String`. We have two instances of this struct. The first, `home`, has
-the value `IpAddrKind::V4` as its `kind` with associated address data of
-`127.0.0.1`. The second instance, `loopback`, has the other variant of
-`IpAddrKind` as its `kind` value, `V6`, and has address `::1` associated with
-it. We’ve used a struct to bundle the `kind` and `address` values together, so
-now the variant is associated with the value.
-
+<span class="caption">Listado 6-1: Almacenamiento de los datos
+    y la variante `IpAddrKind` de una dirección IP usando una `struct`</span>
+
+Aquí, hemos definido una estructura `IpAddr` que tiene dos campos: un `kind` campo
+que es del tipo `IpAddrKind` (la enumeración que definimos previamente) y un `address` campo
+de tipo `String`. Tenemos dos instancias de esta estructura. La primera, `home`, tiene
+el valor `IpAddrKind::V4` como su `kind` con datos de direcciones asociadas de
+`127.0.0.1`. La segunda instancia, `loopback`, tiene la otra variante de
+`IpAddrKind` como su `kind` de valor, `V6`, y tiene la dirección `::1` asociada
+con eso. Hemos usado una estructura para juntar los valores `kind` y
+`address`, por lo que la variante ahora está asociada con el valor.
+
+
+Podemos representar el mismo concepto de una forma más concisa
+usando solo una enumeración en vez de una enumeración como parte
+de una estructura al poner datos directamente en cada variante de
+enumeración. Esta nueva definición de enumeración `IpAddr` dice
+que tanto las variantes `V4` y `V6` tendrán asociados valores `String`:
+=======
 We can represent the same concept in a more concise way using just an enum,
```

Comment:
Please look at this duplicated code after the merge, this is translated just above